### ISSUE TITLE
Method Codecasts\Auth\JWT\Console\KeyGenerateCommand::handle() does not exist  

### ISSUE THREAD

**Issue by CrixuAMG** [codecasts/laravel-jwt#30]
PHP 7.1.6-1~ubuntu16.04.1+deb.sury.org+1 (cli) (built: Jun  9 2017 08:26:34) ( NTS )
Laravel 5.5.11
Laravel-jwt: ^0.8.5

The error occurs when running `php artisan jwt:generate`.

The error: 
```
  [ReflectionException]                                                          
  Method Codecasts\Auth\JWT\Console\KeyGenerateCommand::handle() does not exist 
```

**Comment by hernandev** [comment]
let me check and I'll get back to you.

**Comment by hernandev** [comment]
This was fixed on a PR at that time.

Closing for now, Just checked the source code and that fix is present since version 0.9.0

Thank you
